src/compute-pr-actions.ts

```diff
@@ -305,6 +308,10 @@ export function process(prInfo: PrInfo | BotEnsureRemovedFromProject | BotNoPack
     // Update intro comment
     post({ tag: "welcome", status: createWelcomeComment(info) });
 
+    // Propagate suggestions into actions
+    context.suggestions = noNulls(flatten(info.pkgInfo.map(pkg => pkg.files.map(({ path, suggestion }) =>
+        suggestion && { path, ...suggestion }))));

```

Comment:
This uses `noNulls`, but the list might have `undefined`s.  I see that the function uses `!=` instead of `!==` so it works, but it looks bad to rely on this.  (I remember other places where I wanted to remove undefineds, and missed that it's doing it.)  So it would be better to:
* Rename it to something that indicates `noNullsOrUndefineds` but with some name that isn't so obnoxiously long.  Or maybe just add a comment saying it?
* Or make this code `suggestion ? {...} : null`?

Reply:
Done: #249